packages/acceptance-tests/pkg-tests-specs/sources/commands/workspaces/__snapshots__/foreach.test.js.snap

```diff
@@ -186,6 +186,42 @@ Done
 }
 `;
 
+exports[`Commands workspace foreach should not prefix the output when run with --verbose and --no-label 1`] = `
```

Comment:
This output is a bit dumb, but I suppose should be well-defined. Before this test I had a bug where it was prefixing everything with `null` in this case.